package.json

```diff
@@ -131,8 +131,8 @@
     "yarn-run-all": "^3.1.1"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.35",
-    "@emotion/styled": "^10.0.27",
+    "@emotion/core": "10.0.35",
```

Comment:
Should we peg `@emotion/babel-preset-css-prop` too?

Reply:
I guess yes since it is following same version. I also fixed `eslint-plugin-emotion` for the same reason